test/integrations/validation/Zod.spec.ts

```diff
@@ -0,0 +1,348 @@
+
+import { expect, describe, it } from 'vitest';
+
+import zod from '../../../src/integrations/validation/module';
```

Comment:
In the Comify project, we get everything from the fixtures. This includes the implementation. We've moved to this strategy because some implementations require a (fixed) setup before they can be used.